### ISSUE TITLE
add 'neovim' to default labels if empty

### ISSUE THREAD

**Issue by teto** [nvim-neorocks/luarocks-tag-release#14]
I suppose here : https://github.com/nvim-neorocks/luarocks-tag-release/blob/e3a7af1e6aafed849d01fd3a23aabcd5bdfe2f06/bin/luarocks-tag-release.lua#L168

This way most neovim plugins using this action could eventually come up at https://luarocks.org/labels/neovim
(once https://github.com/luarocks/luarocks-site/issues/188 is fixed).

**Comment by mrcjkb** [comment]
🤔 I want to keep this compatible with non-neovim packages, too. So perhaps just adding `neovim` as a default label would be better - so it can be disabled if unwanted).

**Comment by teto** [comment]
>  I would like to keep this compatible with non-neovim packages

Makes sense, I had not realized that (considerign the orgname maybe "nvim-neorocks" ;)  ). Also the scripts picks up the repository labels, which should probably contain "neovim", so maybe it's just a matter of adding it to the doc or have an heuristic that detects the repo is a neovim plugin and warns the user to set "neovim" as a repo label when missing.
